MonstieWash/Assets/Scripts/Traversal/GameSceneManager.cs

```diff
@@ -186,6 +192,29 @@ public void MoveToScene(string target)
         OnSceneChanged?.Invoke(target);
     }
 
+    /// <summary>
+    /// Swaps the current active scene to a new scene
+    /// </summary>
+    /// <param name="target">The scene to move to.</param>
+    public void MoveToScene(string target, bool targetIsUI)
+    {
+        SetSceneActive(m_currentScene.name, false);
+        SetSceneActive(target, true);
+
+        if (targetIsUI)
+        {
+            InputManager.Instance.SetCursorMode(false);
+            InputManager.Instance.SetControlScheme(InputManager.ControlScheme.MenuActions);
```

Comment:
During testing I noticed that the game still renders the playerhand while navigating to UI scenes, although it doesn't follow the player in them - causing it to remain frozen in place. Maybe playerhand should be disabled for these scenes?
![Screenshot 2024-07-27 003955](https://github.com/user-attachments/assets/d11b2a38-b28b-4287-b6ad-3b0940bcd4b3)

Reply:
Oh yeah that's made for controller - it's a cursor for them to use UI - and I figured since the open day is gonna be played on controller it won't matter how it plays on KB&M for now. In the future we'll have to figure out how to make it follow the real mouse when in use.